src/tests/Autosuggest.js

```diff
@@ -412,6 +426,106 @@ describe('Autosuggest', function() {
     });
   });
 
+  describe('onSuggestionFocused', function() {
+    beforeEach(function() {
+      onSuggestionFocused.mockClear();
+      onSuggestionUnfocused.mockClear();
+      createAutosuggest(
+        <Autosuggest suggestions={getSuburbStrings}
+                     onSuggestionFocused={onSuggestionFocused}
+                     onSuggestionUnfocused={onSuggestionUnfocused} />
```

Comment:
I suggest that in the `onSuggestionFocused` section we don't care about `onSuggestionUnfocused` at all.